ProcessMaker/processmaker
# Bugfix/FOUR-19987: Quick fill > Server Error is displayed when `Save and Reuse filled data` option is selected

## Solution
- Server error message was fixed

## How to Test
- Login PM4
- Create a user
- Create a Group
- Assign the user to group
- Create a simple process (start event - task - task - end event)
- assign the group in the start event
- Assign the group in both tasks
- Login with the new user
- Create a Case
- Fill the screen
- Create a second Case
- Open the task
- Press Create rule button
- Check Save and reuse filled data option
- Check Apply to future tasks option
- Press next option
- Press Quick fill button (Everything must work fine)

## Related Tickets & Packages
https://processmaker.atlassian.net/browse/FOUR-19987

## Code Review Checklist
